OutOfSchool/OutOfSchool.WebApi/Extensions/ControllerExtension.cs

```diff
@@ -1,10 +1,33 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Localization;
+using OutOfSchool.BusinessLogic.Models;
 
 namespace OutOfSchool.WebApi.Extensions;
 
 public static class ControllerExtension
 {
+    /// <summary>
+    /// Extension method, that checks searchResult for null of entities empty and returns result of it.
+    /// </summary>
+    /// <typeparam name="T">Typeof entity.</typeparam>
+    /// <param name="controller">Controller base.</param>
+    /// <param name="searchResult">Instance of SearchResult.</param>
+    /// <returns>
+    /// NoContent() - searchResult is null or his entities is empty.
+    /// Ok() searchResult is not null and his entities is not empty.
+    /// </returns>
+    public static IActionResult GetResultOfGetOperation<T>(this ControllerBase controller, SearchResult<T> searchResult)
```

Comment:
(Test coverage) Could you show me the unit test checking this new method?

Reply:
Now you can find a tests to this method in MapSearchResultToOkOrNoContentExtensionTests.

But, i am not entirely sure, that i done this correctly. Can you check this and write your opinion?